src/status_im2/subs/chat/chats.cljs

```diff
@@ -438,3 +438,13 @@
    [(re-frame/subscribe [:chats/chat (str community-id channel-id)])])
  (fn [[chat]]
    (:muted? chat)))
+
+(re-frame/reg-sub
+ :camera-roll/total-photos-count
+ (fn [{:keys [camera-roll/albums]}]
+   (->> albums
+        :my-albums
+        (reduce
+         (fn [total-album-count curr-album]
+           (+ total-album-count (:count curr-album)))
+         0))))
```

Comment:
This is incorrect. This will not get you the total photos count, but only the total of photos that are classified into albums. The camera roll library we are using does not have an option to get the total photo count. This needs to be done in native code.

Reply:
I am back Saturday and will implement it, Thanks Omar!

Reply:
Done PR here

https://github.com/status-im/react-native-cameraroll/pull/2